plottr/plot/pyqtgraph/autoplot.py

```diff
@@ -361,6 +396,10 @@ class FigureOptions:
     #: how to represent complex data
     complexRepresentation: ComplexRepresentation = ComplexRepresentation.realAndImag
 
+    numAxes: int = 0
+
+    imagData: bool = False
```

Comment:
Should there be comments explaining what they are? they look pretty self explanatory but maybe?